### DISCUSSION TITLE
Unable to customize CSS on Pivot table

### DISCUSSION THREAD

**Discussion by PowerDinosaur** [apache/superset#24883] [Q&A / Help]
Hi guys

I would like to customize the CSS of the pivot table, using the dashboard-chart-id is not good, because it referencing a way bigger area than the actual table, I tried to inspect the HTML of the dashboard and directly editing it in the Firefox inspector I can edit the things i want to like simply put background-color here:
`.css-7tmzyl table.pvtTable {
  position: relative;
  font-size: 12px;
  text-align: left;
  margin: 4px;
  border-collapse: separate;
  font-family: "Inter", Helvetica, Arial;
  line-height: 1.4;
  background-color: brown !important;
}`

However placing background-color part in the css editor does nothing. Can anybody help me out?

Thank you in advance!

Kind regards,

**Comment by fabricio-fn** [comment]
if you make the changes in the browser inspection it will go back to normal when you refresh the page, you will have to use a code editor and make the modifications directly in the css file

**Reply by PowerDinosaur** [reply]
Yes, i know that, however the problem is exactly that If i do this in the code editor nothing happens.

**Reply by fabricio-fn** [reply]
maybe the changes you are making are not being saved

Shortcut to quickly save changes made to code, use CTRL + S command in your css file.

**Reply by fabricio-fn** [reply]
or maybe you didn't put the link of your css file in the html

![Screenshot_2023-08-07_10-29-12](https://github.com/apache/superset/assets/116690430/1ffca9d9-db56-435d-bdf3-a1e059bcaf8e)
![Screenshot_2023-08-07_10-29-27](https://github.com/apache/superset/assets/116690430/973c0bfe-cdf5-4530-abd8-4fc20b822819)

**Comment by rusackas** [comment]
Are you trying to make these changes via a Dashboard CSS template, or to the plugin/component itself?

**Comment by PowerDinosaur** [comment]
Via dashboard css template

**Comment by masgo** [comment]
I ran into the same problem when using the dashboard CSS editor. CSS which affects the dashboard elements, like titles, work fine. CSS which sould affect the pivot table does not work. Something simple like
```
tr {
  background-color: red;
} 
```
has no effects.

The CSS is saved correctly, the CSS is in the HTML code but does not work.

**Comment by rusackas** [comment]
You probably just need to dig into the css inspector and see what's getting
higher precedence. Then you can adjust your selector accordingly. If any
can give a more specific use case or screenshot of what they're hoping to
accomplish, I can try to help.
